resources/etls/LoadApplication.xml

```diff
@@ -144,6 +144,14 @@
             <source schemaName="cds" queryName="ds_bamaantigen" />
             <destination schemaName="cds" queryName="BAMAAntigen" targetOption="truncate" />
         </transform>
+        <transform id="PopulateAntigenPanel" type="org.labkey.di.pipeline.TransformTask">
```

Comment:
I know we are using this pattern of import table -> ds_ query -> target table a lot, but I believe the ds_antigenPanel and ds_virusPanel queries are not necessary since they are identical to their import_ counterparts

Reply:
Now removed, and using the table names instead.